Functions/Gherkin.Tests.ps1

```diff
@@ -60,3 +60,57 @@ Describe 'Invoke-Gherkin' {
         @($gherkin.NamedScenario.PassedScenarios).Count | Should Be 1
     }
 }
+
+$job = Start-Job -ArgumentList $scriptRoot -ScriptBlock {
+    param ($scriptRoot)
+    Get-Module Pester | Remove-Module -Force
+    Import-Module $scriptRoot\Pester.psd1 -Force
+
+    $featureFilePath = "$scriptRoot\Examples\Gherkin\Gherkin-Import.feature"
+    $testsFilePath = "$scriptRoot\Examples\Gherkin\Gherkin-Import.steps.ps1"
+
+    Import-FeatureFile -Path $featureFilePath
+    
+    $props = @{
+        Results              = invoke-gherkin $featureFilePath -PassThru -Show None
+        EntityParamOnlyCount = (Select-String -Path $testsFilePath -SimpleMatch 'param($entity)').count
+        DualParamCount       = (Select-String -Path $testsFilePath -SimpleMatch 'param($entity,$value)').count
+    }
+    
+    $oldContent = Get-Content $featureFilePath
+
+    $newContent = "`r`n`r`n`tScenario: newTeset`r`n`t`tGiven A New Test"
```

Comment:
Using the chars: "\`r\`n" will cause that the code will not be compatible with Linux and macOS. Please use ```$([System.Environment]::NewLine)``` instead.

Reply:
Technically, ``` `r`n``` here won't hurt anything -- it's just going into a .feature file, and neither PowerShell nor Gherkin parsers care about line endings (regardless of OS).

I still think it should be changed to a multi-line string with the line breaks _within_ the string, for readability, but whatever you use needs to be consistent throughout (there are ``` `r`n``` all through this test code).